app/models/mission/MissionTable.scala

```diff
@@ -519,6 +519,17 @@ object MissionTable {
     missions.filter(_.userId === userId.toString).map(_.distanceProgress).sum.run.getOrElse(0F) * METERS_TO_MILES
   }
 
+  /**
+    * Gets distance audited by a user in their current mission, if it exists, in meters.
+    *
+    * @param userId
+    * @return
+    */
+  def getCurrentDistanceAudited(userId: UUID): Float = db.withSession { implicit session =>
```

Comment:
Could you rename the function to be something like `getMetersAuditedInCurrentMission`? I'm also cool with `getDistanceProgressInCurrentMission` or `getMetersProgressInCurrentMission` or something like that. Whatever you can do to make it clear that it is for the current audit mission, and that you are getting the number in meters.